### PR TITLE
feat: Pipeline Updates

### PR DESCRIPTION
**Pull request by barrfalk**
# Description

Update PR pipeline to latest provided by Ministry.

Fixes # (issue)

# How Has This Been Tested?

- [x] Verify cypress tests run
- [x] Manually verified that webeoc components work

## Further comments

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-compliance-enforcement-502-frontend.apps.silver.devops.gov.bc.ca/api/) available
[Frontend](https://nr-compliance-enforcement-502-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge-main.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-compliance-enforcement-502-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-compliance-enforcement-502-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-compliance-enforcement-502-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-compliance-enforcement-502-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge.yml)

### CODE REVIEW

**Review by afwilcox — COMMENTED** [review]
Are the

File: charts/app/templates/backend/templates/deployment.yaml
[the remark this answers is not in the document]
Comment (barrfalk) [reply charts/app/templates/backend/templates/deployment.yaml:1]: Good call, I've added that in and the pipeline appears to still work.

File: .github/workflows/analysis.yml
[the remark this answers is not in the document]
Comment (barrfalk) [reply .github/workflows/analysis.yml:1]: CodeQL is now executed by default, no need for a GitHub action.

image

File: build/asset-manifest.json
[the remark this answers is not in the document]
Comment (barrfalk) [reply build/asset-manifest.json:1]: Nice find.  To be honest, I don't know how I added that, it must have been by mistake.  I've removed it.

**Review by afwilcox — APPROVED** [review]

**Pull request merged**